.gitignore

```diff
@@ -1,2 +1,5 @@
 /target
 **/*.rs.bk
+
+# IntelliJ project directory
+.idea
```

Comment:
I prefer to add my IDE-specific project files to `.git/info/ignore` rather than the version controlled `.gitignore`.

Reply:
Will fix in next MR